### DISCUSSION TITLE
weawer wallet transaction fees

### DISCUSSION THREAD

**Discussion by Akrepkapo** [IBAX-io/go-ibax#2065] [Ideas] [answered]
Hello dear team. The simplest operation here, even editing first and last name, requires hundreds or even thousands of ibxc coins per additional letter and character. And even if we calculate it from ibxc's pre-sale price, it means hundreds of dollars in cost. I think this is a mistake that needs to be corrected.i think weaver wallet costs and simple transactions should have very low transaction fees
![Uploading Screenshot_2022-10-26-01-10-20-32_40deb401b9ffe8e1df2f1cc5ba480b12.jpg…]()

**Comment by yolandadadada** [comment, marked as answer]
This is the normal deduction of fees
